lib/wsdl.js

```diff
@@ -1558,7 +1558,11 @@ WSDL.prototype.objectToXML = function(obj, name, namespace, xmlns, first, xmlnsA
       }
 
       if (!Array.isArray(child)) {
-        parts.push(['<', ns, name, attr, xmlnsAttrib, '>'].join(''));
+        var insNull = '';
+        if(typeof child === 'object' && child === null) {
```

Comment:
Reconsidering, let me test if it would produce any false results if we check only `child === null`